plugins/airtable/src/data.ts

```diff
@@ -64,6 +65,11 @@ export async function getTables(baseId: string, signal: AbortSignal): Promise<Ai
 const EMAIL_REGEX = /\S[^\s@]*@\S+\.\S+/
 const PHONE_REGEX = /^(\+?[0-9])[0-9]{7,14}$/
 
+const ArrayLikeSchema = v.pipe(
+    v.array(v.object({ type: v.optional(v.string()), id: v.string(), url: v.string() })),
+    v.minLength(1)
+)
```

Comment:
Better name welcome! Didn't want to think about it for too long.

Reply:
> It replaces if (value.length === 0) return null in both spots below, is that not right?

ah yes true. I mean it more in the sense that this is a schema for validating API structure. And if you'd use it to validate an entire API response you wouldn't want it to be invalid without a length. 

But thats theoritical and a nitpick and I should have marked as such. More as a comment because it can be an easy trap to use the same schema for full API response validation and then it'd be incorrect.

Reply:
Ah, yeah, that's why I went extra verbose with the schema names, hope that's enough of a deterrent! 🤞